docs/user/adding-glean-to-your-project.md

```diff
@@ -306,23 +306,23 @@ Please refer to the [custom pings documentation](pings/custom.md).
 
 ### Parallelism
 
-All of Glean's target languages use a separate worker thread to do most of Glean's work, including any I/O. This thread is fully managed by Glean as an implementation detail. Therefore, users should be free to use the Glean API wherever it is most convenient, without worrying about the performance impact of updating metrics and sending pings.
+All of the Glean SDK's target languages use a separate worker thread to do most of its work, including any I/O. This thread is fully managed by the Glean SDK as an implementation detail. Therefore, users should feel free to use the Glean SDK wherever it is most convenient, without worrying about the performance impact of updating metrics and sending pings.
 
 {{#include ../tab_header.md}}
 
 <div data-lang="Python" class="tab">
-Since Glean performs disk and networking I/O, it tries to do as much of its work as possible on separate threads and processes.
+Since the Glean SDK performs disk and networking I/O, it tries to do as much of its work as possible on separate threads and processes.
 Since there are complex trade-offs and corner cases to support Python parallelism, it is hard to design a one-size-fits-all approach.
 
 #### Default behavior
 
-When using the Python bindings, most of Glean's work is done on a separate thread, managed by Glean itself.
-Glean releases the Global Interpreter Lock (GIL), therefore your application's threads should not be in contention with Glean's thread.
+When using the Python bindings, most of the Glean SDK's work is done on a separate thread, managed by the Glean SDK itself.
+The Glean SDK releases the Global Interpreter Lock (GIL) for most of its operations, therefore your application's threads should not be in contention with the Glean SDK's worker thread.
 
-Glean installs an [`atexit` handler](https://docs.python.org/3/library/atexit.html) so the Glean thread can cleanly finish when your application exits.
+The Glean SDK installs an [`atexit` handler](https://docs.python.org/3/library/atexit.html) so that its worker thread can cleanly finish when your application exits.
 This handler will wait up to 30 seconds for any pending work to complete.
 
-In addition, by default ping uploading is performed in a separate child process. This process will continue to upload any pending pings even after the main process shuts down. This is important for commandline tools where you want to return control to the shell as soon as possible and not be delayed by network connectivity.
+In addition, by default, ping uploading is performed in a separate child process. This process will continue to upload any pending pings even after the main process shuts down. This is important for commandline tools where you want to return control to the shell as soon as possible and not be delayed by network connectivity.
```

Comment:
English nit, the two subordinate phrases starting this sentence seems awkward "In addition, by default...".  Can we just drop the "In addition"?

Reply:
Good point.